src/Synth/OscilGen.cpp

```diff
@@ -595,7 +601,16 @@ void OscilGen::recalculateDefaultWaveTable(WaveTable * wt, int Presonance, bool
     // abuse pointer swap to steal memory from current wavetable
     wt->insert(data, freqs, semantics, true);
 
-    calculateWaveTableTensors(freqs, semantics, data, Presonance, fillWithZeroes);
+    calculateWaveTableScales(freqs, semantics, fillWithZeroes);
+    for(std::size_t i = 0; i < semantics.capacity(); ++i)
```

Comment:
Ok, this is pretty awesome :) This function would have to be invoked only on a default oscilgen though, but I'm pretty sure that's the intent.

Reply:
Not sure what you mean by default OscilGen. It's only used when defaulting ADnote.

Now that you say it, I have never tested loading savefiles with this whole branch. I'm going to do a few tests...